plugins/connection/netconf.py

```diff
@@ -141,6 +141,15 @@
     - name: ANSIBLE_PERSISTENT_CONNECT_TIMEOUT
     vars:
     - name: ansible_connect_timeout
+  proxy_command:
+    default: ''
+    description:
+      - Proxy information for running the connection via a jumphost.
+      - This requires ncclient >= 0.6.10 to be installed on the controller.
+    env:
+      - name: ANSIBLE_NETCONF_PROXY_COMMAND
+    ini:
+      - {key: proxy_command, section: paramiko_connection}
```

Comment:
I'm not sure if ssh is the right namespace for a common var or if paramiko would be better suited given why we need it.
```suggestion
      - {key: proxy_command, section: paramiko_connection}
      vars:
      - name: ansible_ssh_proxy_command
      - name: ansible_netconf_proxy_command
```